CREDITS-MUSIC

```diff
@@ -80,7 +80,7 @@ Cluster 2
 	MAP17: "Mow Them Down!" by Lola "BlueWorrior" Harvey
 	MAP18: "Realistic Approach" by Kevin "Velvetic" Martins
 	MAP19: "Deceit" by Julian "Julian Hope" Aubourg, modified by m
-	MAP20: dave3d05.mid by jute
+	MAP20: "Beasts of THe Horizon" by Josephus "DH4050" Astartes
```

Comment:
"THe"?

Reply:
oops XD anyways, im glad you enjoyed it! ^^

Reply:
alright, i've fixed it. ^^